gettsim/tests/test_wg.py

```diff
@@ -62,16 +66,20 @@ def test_wg(year):
     for col in out_cols:
         df[col] = np.nan
     df = df.groupby("hid").apply(wg, tb=tb)
-    expected = load_test_data(year, file_name, test_column)
-    assert_frame_equal(
-        df[test_column], expected, check_exact=False, check_less_precise=2
-    )
+    assert_frame_equal(df[test_column], year_data[test_column])
+
+
+@pytest.fixture
```

Comment:
This fixture is not necessary. Just reuse the other one.

Reply:
True. But there is a second csv for test_wg. Will ask Eric why its there

Reply:
Sorry. The mistake was to not use the second csv. Now its right!